fastapi_injector/attach.py

```diff
@@ -1,12 +1,26 @@
 from fastapi import FastAPI
-from injector import Injector
+from injector import Injector, InstanceProvider, singleton
 
 from fastapi_injector.exceptions import InjectorNotAttached
+from fastapi_injector.request_scope import RequestScopeOptions
 
 
-def attach_injector(app: FastAPI, injector: Injector) -> None:
-    """Call this function on app startup to attach an injector to the app."""
+def attach_injector(app: FastAPI, injector: Injector, **kwargs) -> None:
```

Comment:
Related to my other comment, maybe this function could accept an argument of type `RequestScopeOptions` with a default. I think it would make the signature more transparent.

Reply:
Done - I did originally consider this but I wasn't sure which style you would prefer :)